src/components/dialog/bl-dialog.css

```diff
@@ -1,36 +1,52 @@
-dialog {
+.dialog,
+.dialog-polyfill .container {
   padding: 0;
   border: 0;
   background: var(--bl-color-primary-background);
   border-radius: var(--bl-border-radius-l);
   max-width: calc(100vw - var(--bl-size-4xl));
   max-height: calc(100vh - var(--bl-size-4xl));
+  min-width: 424px;
+  min-height: 178px;
   overflow: hidden;
 }
 
-dialog::backdrop {
+.dialog::backdrop {
   background: #273142;
   opacity: 0.7;
 }
 
-dialog .container {
-  min-width: 424px;
-  min-height: 178px;
+.dialog-polyfill {
+  display: var(--display);
+  position: fixed;
+  left: 0;
+  top: 0;
+  min-width: 100vw;
+  min-height: 100vh;
```

Comment:
I think we should just set width and height, not minimum values. Dialog background should not be scrollable, right?

Reply:
Yes you are right, setting min-max value is enough for this. I actually fixed it here, but it may have come back with ctrl+z 🥲